public/app/core/components/AppChrome/ReturnToPrevious/ReturnToPreviousWrapper.tsx

```diff
@@ -0,0 +1,49 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+import { useQueryParams } from 'app/core/hooks/useQueryParams';
+
+import { ReturnToPrevious } from './ReturnToPrevious';
+
+export const ReturnToPreviousWrapper = () => {
+  const params = useQueryParams()[0];
+  const [paramsExist, setParamsExist] = React.useState(params?.returnToTitle && params?.returnToUrl);
```

Comment:
could be good to use `__returnToTitle`  and `__returnToUrl` to lessen risk of conflict with url params used by the page.

Reply:
I'm not worried about it working when a new tab is open, so if that makes things better, works for me!